### PR TITLE
feat: User on app can filter auction results by auction house

### PR DESCRIPTION
**Pull request by olerichter00**
The type of this PR is: Feature

This PR resolves [CX-848]

### Description

Adds an "auction house" filter for auction results on the artist insights tab.

| | | | 
|---|---|---|
| Simulator Screen Shot - iPhone 8 - 2021-06-22 at 13 02 27 | Simulator Screen Shot - iPhone 8 - 2021-06-22 at 13 02 31 | Simulator Screen Shot - iPhone 8 - 2021-06-22 at 13 02 36 |

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-848]: https://artsyproduct.atlassian.net/browse/CX-848

### CODE REVIEW

File: src/lib/Components/ArtworkFilter/ArtworkFilterHelpers.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ export enum FilterDisplayName {
   locationCities = "Artwork location",
   materialsTerms = "Material",
   medium = "Medium",
+  organizations = "Auction house",
```
Comment (olerichter00) [inline src/lib/Components/ArtworkFilter/ArtworkFilterHelpers.ts:19]: Note: The display name of the filter is "Auction house" but the filter is called organizations when we query for auction results.

File: src/lib/Components/ArtworkFilter/Filters/__tests__/AuctionHouseOptions-tests.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      total: null,
+      followedArtists: null,
+    },
+  }
+
+  it("selects only the option that is selected", () => {
+    const tree = renderWithWrappers(<MockAuctionHouseScreen {...getEssentialProps()} initialData={initialState} />)
+
+    // selected auction house index
+    const selectedAuctionHouseIndex = Math.floor(Math.random() * Math.floor(AUCTION_HOUSE_OPTIONS.length))
```
Comment (MounirDhahri) [inline src/lib/Components/ArtworkFilter/Filters/__tests__/AuctionHouseOptions-tests.tsx:37]: **praise:**  nice tests

**Review by MounirDhahri — APPROVED** [review]

**Comment by MounirDhahri** [comment]
#squashongreen

File: src/lib/Components/ArtworkFilter/Filters/__tests__/AuctionHouseOptions-tests.tsx (hunk 2 above)
Comment (olerichter00) [reply src/lib/Components/ArtworkFilter/Filters/__tests__/AuctionHouseOptions-tests.tsx:37]: Thanks! Actually, it's copy-pasted from a test you wrote a few months ago 🎉

**Pull request merged**